WordPress/src/main/java/org/wordpress/android/ui/reader/ReaderPostDetailFragment.java

```diff
@@ -69,10 +75,14 @@
                    ReaderCustomViewListener,
                    ReaderWebViewPageFinishedListener,
                    ReaderWebViewUrlClickListener {
+    private static final int INTENT_WELCOME = 0;
```

Comment:
`INTENT_WELCOME` is defined in both `DeepLinkingIntentReceiverActivity` and `ReaderPostDetailFragment`. How about moving it to <a href="https://github.com/wordpress-mobile/WordPress-Android/blob/develop/WordPress/src/main/java/org/wordpress/android/ui/RequestCodes.java">RequestCodes</a>?